### ISSUE TITLE
ArangoError 1208: illegal name

### ISSUE THREAD

**Issue by arturasjurev** [arangodb/arangodb#2949]
Hello, so I ran into a problem with starting arangorestore after upgrading arango. 

Im using arango on one of my docker containers and recently ive upgraded arango from 2.8 to 3.2.
My arangodb conf : 
```
# ArangoDB configuration file
# 
# Documentation: 
# https://docs.arangodb.com/ConfigureArango/README.html
# 

[database]
directory = /home/proj/shared
# directory = @HOMEDRIVE@/@HOMEPATH@/arangodb/databases

maximal-journal-size = 33554432

[server]
# Specify the endpoint for HTTP requests by clients.
#  tcp://ipv4-address:port
#  tcp://[ipv6-address]:port
#  ssl://ipv4-address:port
#  ssl://[ipv6-address]:port
#  unix:///path/to/socket
#
# Examples:
#   endpoint = tcp://0.0.0.0:8529
#   endpoint = tcp://127.0.0.1:8529
#   endpoint = tcp://localhost:8529
#   endpoint = tcp://myserver.arangodb.com:8529
#   endpoint = tcp://[::]:8529
#   endpoint = tcp://[fe80::21a:5df1:aede:98cf]:8529
#
endpoint = tcp://0.0.0.0:8529

# resuse a port on restart or wait until it is freed by the operating system
# reuse-address = false

# disable authentication for the admin frontend
authentication = false

# number of server threads
threads = 4

# the user and group are normally set in the start script
# uid = arangodb
# gid = arangodb

[scheduler]

# number of threads used for I/O

[javascript]
startup-directory = /usr/share/arangodb3/js
app-path = /var/lib/arangodb3-apps
# app-path = @HOMEDRIVE@/@HOMEPATH@/arangodb/apps

# number of worker threads for V8
v8-contexts = 5

[log]
level = info
file = -

[cluster]
#disable-dispatcher-kickstarter = yes
#disable-dispatcher-frontend = yes
data-path = /home/proj/shared/cluster
log-path = /home/proj/shared/cluster
agency-prefix = /usr/lib/arangodb/etcd-arango
arangod-path = /usr/sbin/arangod
dbserver-config = /home/proj/shared/arangod.conf
```

So i run : 
`arangorestore  --include-system-collections true --server.database $dbname --create-database true --input-directory $location` 

Error i get :
```
Restoring 27240629947776
Please specify a password:  

Creating database 'org_27240629947776'
Server version: 3.2.0
# Connected to ArangoDB 'http+tcp://127.0.0.1:8529'
# Re-creating document collection '_aqlfunctions'...
2017-08-04T06:20:06Z [56] ERROR got error from server: HTTP 500 (Internal Server Error): ArangoError 1208: illegal name
Processed 0 collection(s), read 0 byte(s) from datafiles, sent 0 batch(es)
Restoring _sys
Please specify a password: 

Server version: 3.2.0
# Connected to ArangoDB 'http+tcp://127.0.0.1:8529'
# Re-creating document collection '_aqlfunctions'...
2017-08-04T06:20:09Z [59] ERROR got error from server: HTTP 500 (Internal Server Error): ArangoError 1208: illegal name
Processed 0 collection(s), read 0 byte(s) from datafiles, sent 0 batch(es)
Restoring system
Please specify a password: 

Creating database 'my_system'
Server version: 3.2.0
# Connected to ArangoDB 'http+tcp://127.0.0.1:8529'
# Re-creating document collection '_aqlfunctions'...
2017-08-04T06:20:09Z [60] ERROR got error from server: HTTP 500 (Internal Server Error): ArangoError 1208: illegal name
Processed 0 collection(s), read 0 byte(s) from datafiles, sent 0 batch(es)

```

Thanks in advance !

**Comment by arturasjurev** [comment]
By the way ,  I'm currently using python-arango lib is v2.2.0 and willing to upgrade to v3.6 after successfull arango upgrade.

**Comment by jsteemann** [comment]
Fixed in 3.2.1

**Comment by jsteemann** [comment]
3.2.1 has been released and is [available for download](https://www.arangodb.com/download-major/).
